### PR TITLE
Added a middleware to capture invalid accesspackage object

### PR DESCRIPTION
**Pull request by acn-dgopa**
## Related Issue(s)
- #{issue number}

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

## Summary by CodeRabbit

* **New Features**
  * Implemented validation on authentication endpoints to reject requests with invalid access package structures containing multiple identifiers. Users will receive detailed error messages when submitting malformed requests.

* **Tests**
  * Added comprehensive unit test coverage for the new validation feature, including edge cases and various request scenarios.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: src/Authentication/Middlewares/RawJsonInspectionMiddleware.cs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            bool isTargetEndpoint =
+                context.Request.Path.StartsWithSegments("/authentication/api/v1/systemregister/vendor", StringComparison.OrdinalIgnoreCase) ||
+                context.Request.Path.Value.Contains("/accesspackages", StringComparison.OrdinalIgnoreCase);
+
+            if (context.Request.ContentType?.Contains("application/json") == true &&
+                context.Request.Method is "POST" or "PUT" && isTargetEndpoint)
+            {
+                context.Request.EnableBuffering();
+
+                using var reader = new StreamReader(
```
Comment (Alxandr) [inline src/Authentication/Middlewares/RawJsonInspectionMiddleware.cs]: Middleware should be **very** careful about reading the request body as this might have implications on later middleware (like MVC).
